samples/msgext-ai-sentiment-analysis/nodejs/.vscode/launch.json

```diff
@@ -0,0 +1,73 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch App (Edge)",
+            "type": "msedge",
+            "request": "launch",
+            "url": "https://teams.microsoft.com/l/app/${{local:TEAMS_APP_ID}}?installAppPackage=true&webjoin=true&${account-hint}",
```

Comment:
Please check if we need to push this file?

Reply:
Generated by default and, it was there in your PR as well.

Reply:
ok